pkg/auth/authenticator/password/htpasswd/htpasswd.go

```diff
@@ -136,7 +139,7 @@ func testPassword(password, hash string) (bool, error) {
 	case strings.HasPrefix(hash, "{SHA}"):
 		// SHA-1, insecure
 		return testSHAPassword(password, hash[5:])
-	case len(hash) == 13:
+	case len(hash) == 13 || strings.HasPrefix(hash, "$"):
```

Comment:
Done. Although obviously not future proof ... I agree that these things don't change every day.